### PR TITLE
chore: Small README update

### PR DESCRIPTION
**Pull request by Soap-141**
GitHub Issue: #

## Proposed Changes

- Documentation content changes

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [x] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

**Review by GabrielChouinardLetourneau — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -41,14 +40,14 @@ A small sample is available as a playground.
 
 An example of instantiation as follow with the fallback to the pin code with some text descriptions to display for the user.
 
-```
+``` cs
```
Comment (jeanplevesque) [inline README.md:43]: does that work?
It think it's usually ```csharp

File: README.md [hunk 2]
```diff
@@ -21,13 +21,12 @@ The biometryService Interface 'IBiometryService' Implement the following method
...
 As of now, this is the list of features available per platform.
 
-
-| Feature          | iOS | Android | UWP  |
-| ---------------- | --- | ------- | ---- |
-| GetCapability    | x   | x       | x    |
-| ValidateIdentity | x   | x       | Mock |
-| Encrypt          | x   | x       | Mock |
-| Decrypt          | x   | x       | Mock |
+| Feature          | iOS | Android | UWP  | WinUI |
```
Comment (jeanplevesque) [inline README.md:24]: I think you could split iOS (Xamarin) and iOS (MAUI), same with Android

File: README.md [hunk 3]
```diff
@@ -100,12 +108,16 @@ It will return a struct `BiometryCapabilities` with the detailled device configu
 
 On Android depending on how the service is instantiated, some features might not be available and will throw errors.
 
+---
```
Comment (jeanplevesque) [inline README.md:111]: no need for separators, we already have the headers

File: README.md (hunk 1 above)
Comment (Soap-141) [reply README.md:43]: It works, I used the preview feature on VSCode.

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**